drivers/hmis_csv_importer/app/models/hmis_csv_importer/import_override.rb

```diff
@@ -19,6 +21,10 @@ class HmisCsvImporter::ImportOverride < GrdaWarehouseBase
     order(:file_name, :replaces_column, :matched_hud_key)
   end
 
+  scope :expired, -> do
+    where(expires_on: ..Time.zone.today.yesterday)
```

Comment:
Is time zone needed? Also if the intention is to filter everything the expired before today, maybe the `...` operator is clearer? 
```suggestion
    # records where expires_on < today
    where(expires_on: ...Date.today)
```

Reply:
Thanks!